### PR TITLE
chore(readme): update contributing with astexplorer tool

### PR DESCRIPTION
**Pull request by MrFix93**
As discussed with @nicojs , ASTExplorer should be mentioned in the CONTRIBUTING file as it's very convenient to use while creating new features.

### CODE REVIEW

**Comment by kmdrGroch** [comment]
Hey, thanks for PR, but why is it changed exactly? I see * to - and * to __ changes

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ New features are welcome! Either as requests or proposals.
 Don't get discouraged! We estimate that the response time from the
 maintainers is around a day or so.
 
+## Tools
+
+As Stryker uses the Abstract Syntax Tree (AST) to create mutations, you recommend you to use [ASTExplorer](https://astexplorer.net/) to get familiar with the concepts involved.
```
Comment (kmdrGroch) [inline CONTRIBUTING.md]: we recommend you? Or it is recommended. Or make two sentences.
Stryker uses the Abstract Syntax Tree (AST) to create mutations.
If you want to learn more about it, check out [ASTExplorer](https://astexplorer.net/) to get familiar with the concepts involved.

What do you think about it? :)

Comment (MrFix93) [reply CONTRIBUTING.md]: See the new commit, I changed it already ;)

Comment (kmdrGroch) [reply CONTRIBUTING.md]: Well then for me it's good. I am just thinking about 'concepts involved' or 'involved concepts', but  I guess both are correct

**Comment by nicojs** [comment]
Great job @MrFix93 and @kmdrGroch!

**Pull request merged**